install.sh

```diff
@@ -255,12 +255,27 @@ joinmarket_install ()
 {
     jm_pkgs=( 'jmbase' 'jmdaemon' 'jmbitcoin' 'jmclient' )
     for pkg in ${jm_pkgs[@]}; do
+        pip uninstall -y "${pkg/jm/joinmarket}"
```

Comment:
Much cleaner and less dangerous than the previous `rm` commands.  It even removes the path strings from `easy-install.pth` as it goes.

Reply:
@fivepiece WDYM by the previous `rm` commands? Don't see anything in the diff here... thanks.

Reply:
@mecampbellsoup it's in the previous commit:
https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/90/commits/175ca277411266055a3dbd1e5ee8b299fd5c4b7a